src/states/game/card_drafting.gd

```diff
@@ -72,4 +74,7 @@ func select_card(selection: DualCardData) -> void:
 		draft_round = 1
 		Global.card_replay_moves = Global.card_current_moves
 		Global.card_current_moves = {}
+		$CenterContainer.hide()
+		modulate = Color(0.0, 0.0, 0.0, 0.0) # Look like we're hiding but prevent the mouse from passing through
```

Comment:
Is there a constant for `Color.TRANSPARENT` or something?

Reply:
Good catch